manager/assets/modext/workspace/namespace/modx.namespace.panel.js

```diff
@@ -77,7 +77,7 @@ MODx.grid.Namespace = function(config) {
             ,editor: { xtype: 'textfield' }
         }]
         ,tbar: [{
-            text: _('create_new')
+            text: _('namespace_create')
```

Comment:
You can't rename the lexicon key here without renaming it in the lexicon files.

Reply:
This entry is on the way `core/lexicon/en/propertyset.inc.php` - in 32 line. But she was not used